store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9539,4 +9539,9 @@ TODO: delete them permanently from here
 <attr id="3018" name="zimbraPreviousEphemeralBackendURL" type="string" cardinality="single" optionalIn="globalConfig" immutable="1" since="8.8.1">
   <desc>URL of the previous ephemeral storage backend</desc>
 </attr>
+
+<attr id="3019" name="zimbraAttributeMigrationInfo" type="string" cardinality="single" optionalIn="globalConfig" immutable="1" since="8.8.1">
```

Comment:
Sorry, didn't read the full description properly.  Clearly can't be an enum.  On the immutable issue.  I think there really is a bug but it can probably be addresses separately.  Just creating a COS after a SOAP test run barfs.  I'm guessing because we clone the default COS and if it has 2 factor auth attributes in it, we just blindly copy them over even though they probably don't make sense.